### ISSUE TITLE
Error: Unable to Determine Encoding for New Files

### ISSUE THREAD

**Issue by lljbash** [zbirenbaum/copilot.lua#351]
After #336 was merged, creating a new file in Neovim and attempting to complete with Copilot triggers the following error, disrupting the completion process:

```
Error executing vim.schedule lua callback: ...oad/nvim-linux64/share/nvim/runtime/lua/vim/lsp/util.lua:178: Invalid encoding: ""
stack traceback:
	[C]: in function 'error'
	...oad/nvim-linux64/share/nvim/runtime/lua/vim/lsp/util.lua:178: in function 'get_line_byte_from_position'
	...oad/nvim-linux64/share/nvim/runtime/lua/vim/lsp/util.lua:485: in function 'apply_text_edits'
	...l/share/nvim/lazy/copilot.lua/lua/copilot/suggestion.lua:491: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>
```

`set encoding?` shows `encoding=utf-8`. Closing and reopening the file resolves the issue.

NVIM v0.10.2
Build type: Release
LuaJIT 2.1.1713484068

**Comment by powerLEO101** [comment]
I am meeting the same issue, but closing and reopening the file does not resolves for me.

**Comment by mars90226** [comment]
I add the following code to use `'encoding'` as default encoding to fix this:
```
	if encoding == "" then
		encoding = vim.api.nvim_get_option_value('encoding', {})
	end
```
Put these below the line 490 of `suggession.lua` file to fix the problem.

**Comment by Syu-fu** [comment]
This problem is already fixed in #346

**Comment by lljbash** [comment]
> This problem is already fixed in #346

Oh, thanks for the fix! I hope this gets merged as soon as possible.

**Comment by zbirenbaum** [comment]
Fixed via #346, thanks @Syu-fu!
